keel-api/src/test/resources/examples/titus-cluster-example.yml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: titus.spinnaker.netflix.com/v1
+kind: titus-cluster
```

Comment:
thoughts on making this just `cluster` like for EC2? we've already got the cloud provider encoded into `apiVersion`, and having a consistent `kind` across providers seems more along the lines of how we've historically modeled these things in both the API + UI

Reply:
Can't because of https://github.com/spinnaker/keel/issues/527, but I will fix that in the future to do just that

Reply:
oh hey that's great, ignore me then.

Reply:
I had a similar question: even for the `apiVersion`, could we perhaps be shooting for a `containers.spinnaker.netflix/v1`, as opposed to a Titus-specific set of objects? My feeling is that a spec for Titus would be largely the same as for Kubernetes or ECS. Then we could use annotations to specify the container runtime (see the notion of a Kubernetes [Ingress "class"](https://kubernetes.github.io/ingress-nginx/user-guide/multiple-ingress/) for an example), and for any runtime-specific properties.

Reply:
I think I want to keep titus separate until we have a goal to make titus and kubernetes integration look the same. I think there's little value in using the kubernetes model right now . To me, titus feels more like ec2 than kuberentes.  This would be an interesting UX question to explore later.

Reply:
OK... To be clear, though, what I meant was not to use the Kubernetes model at all, but to generalize the model for all container-based runtimes so we don't have to implement a largely redundant set of classes for each one of them. Looking at the spec the way you have it defined right now, we could easily translate that to both Titus objects, Kubernetes objects, AWS ECS objects, etc. under the covers while using the same higher-level API objects for keel. Hope that makes sense.

Reply:
I agree. One thing I saw as I was creating this PR was that I had to make new "cluster" objects (because they weren't shared outside of the ec2 plugin). I'm trying to think about how we pull the models out into `keel-api` while still allowing them to be "customized" for each resource. I haven't done that in either of my titus PRs because I'd like that to be a stand alone change. It's going to be kind of tricky I think.